KWVerificationCodeView/Classes/KWVerificationCodeView.swift

```diff
@@ -168,7 +156,25 @@ public protocol KWVerificationCodeViewDelegate: class {
 
     return verificationCode
   }
-
+    
+    public func ClearVerificationCode(){
+        
+        for (index, textField) in textFieldViews.enumerated() {
+            textField.numberTextField.text = ""
+            if(index == 0)
+            {
+                textField.numberTextField.becomeFirstResponder()
+            }
+        }
+    }
+    
+    public func setverfiationCode(code : String)
+  {
```

Comment:
Please use this bracket after the function name. Please use consistent code style.